packages/create-sitecore-jss/src/initializers/angular-sxp/index.ts

```diff
@@ -0,0 +1,35 @@
+import path, { sep } from 'path';
```

Comment:
Not related to angular-sxp initializer directly, but: please modify the default angular initalizer to include angular-sxp ( addInitializers.push('angular-sxp')) by defult for now. this will ensure team city will not fail, until we implement the proper sxp/xmcloud separation in create-sitecore-jss.

Reply:
yea that is interesting suggestion, we can do that too. The plan was to modify the teamcity commands to initialize app with 'angular,angular-sxp'. the same goes for the developing tests in automated tests repo (which is already done in a new branch)

Reply:
@art-alexeyenko @yavorsk It's expected that automation tests will be updated after this change